README.md

```diff
@@ -15,68 +13,62 @@ Anaconda works out of the box but there are multitude of options and features th
 * [Linting theme customization](http://damnwidget.github.io/anaconda/IDE/#toc_50)
 * [Using Vagrant Environments](http://damnwidget.github.io/anaconda/vagrant/)
 
-# License
-
-This program is distributed under the terms of the GNU GPL v3. See the [LICENSE](https://raw.github.com/DamnWidget/anaconda/master/LICENSE) file for details.
-
-# Tooltips
-
-I know lot of you asked for this already, anaconda officially supports user themeable
-tooltips and advanced displaying signatures. This is how it looks in a dark theme:
-![](http://damnwidget.github.io/anaconda/img/tooltips.png)
-
-**note**: this feature is enabled for users of Sublime Text 3 build 3070 or superior only.
-
-# Troubleshootings
+## License
+This program is distributed under the terms of the GNU GPL v3. See the [LICENSE][license] file for more details.
 
-Guide to solve common issues
+## Tooltips
+anaconda officially supports user themeable tooltips and displaying advanced signatures. This is how it looks in a dark theme:
 
-## Anaconda does not appear in the available packages list on Package Control
+![Tooltips Image][tooltips-dark-image]
 
-You have to update your package control version to the version 2.0 or better
+**Note**: This feature is enabled for users of Sublime Text 3 build 3070 or superior only.
 
-## I get errors in the console about "the file can't be open" in worker.py file
+## Troubleshooting
+This section lists some common issues faced by users, along with workarounds.
 
-Your sublime text can't find the interpreter that you set in your configuration, by default, anaconda set this as `python` so it will get your configured Python interpreter in your PATH (if any)
+#### Anaconda does not appear in the Available Packages list on Package Control.
+**Work-around**: You have to update your Package Control version to the version 2.0 or better.
 
-## Autocomplete for import behaves badly
+#### Errors in the console about "the file can't be open" in worker.py file.
+Your Sublime Text can't find the interpreter that you set in your configuration, by default, anaconda set this as `python` so it will get your configured Python interpreter in your PATH (if any).
+**Work-around**: Add a Python interpreter (named `python`) to your PATH.
```

Comment:
I think this one should be more like:
Set the value of the `python_interpreter` to a valid Python interpreter path like for example `/usr/local/bin/python3.4`